internal/audit/audit_test.go

```
@@ -0,0 +1,111 @@
+
+	// Append a malformed line directly (writers other than us could in
+	// theory corrupt the file).
+	path, _ := LogPath()
+	f, err := os.OpenFile(path, os.O_APPEND|os.O_WRONLY, 0o600)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if _, err := f.Write([]byte("not json\n")); err != nil {
+		t.Fatal(err)
+	}
+	f.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `f.Close` is not checked

(errcheck)